roles/forklift/tasks/rebuild.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: 'vagrant {{ item }} boxes'
```

Comment:
you can't use `{{ item }}` in the `name` as that is generated outside of the loop. so propbably just `rebuild vagrant boxes` or so

Reply:
Updated